core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -571,7 +571,12 @@ class SparkContext(config: SparkConf) extends Logging {
     _shutdownHookRef = ShutdownHookManager.addShutdownHook(
       ShutdownHookManager.SPARK_CONTEXT_SHUTDOWN_PRIORITY) { () =>
       logInfo("Invoking stop() from shutdown hook")
-      stop()
+      try {
+        stop()
+      } catch {
+        case e: Throwable =>
+          logWarning("Ignoring Exception while stoping SparkContext. Exception: " + e)
```

Comment:
`stoping` -> `stopping`

Reply:
use this format 
`logWarning("", exception)`

Reply:
@felixcheung Thanks for the comments. I updated it.